e2e/specs/multipage_apps.spec.js

```diff
@@ -96,4 +96,36 @@ describe("multipage apps", () => {
     // Expand the nav again
     cy.get('[data-testid="stSidebarNavSeparator"] svg').click();
   });
+
+  describe("handles st.switch_page", () => {
```

Comment:
How hard would it be convert this to Playwright? Didn't we make a rule that changes should be in Playwright?

At the minimum, can we make these tests be in playwright?